src/pages/IndividualArticle.jsx

```diff
@@ -17,7 +21,25 @@ const IndividualArticle = () => {
         }).catch((err) => {
             setError({ err });
         })
-    }, [article_id, article])
+    }, [article_id])
+
+    const handleVote = (vote) => {
+        if(hasVoted) alert('You have already voted on this article');
+
+        if(!hasVoted) {
```

Comment:
it's better practice to have an else statement if there is a behaviour you would like to happen, if an initial condition has not been met